config/generic-bigtreetech-skr-mini-e3-v3.0.cfg

```diff
@@ -107,8 +107,26 @@ pin: PB15
 [fan]
 pin: PC6
 
-[filament_switch_sensor e0_sensor]
-switch_pin: PC15
+# If using a Filament Runout Sensor, uncomment the two lines below
+#[filament_switch_sensor e0_sensor]
+#switch_pin: PC15
+
+# If using bltouch or similar, uncomment and edit the lines below
+#[bltouch]                            
```

Comment:
Please only add basic options for bltouch, some of these are only to be used with certain probes, like clones. This is a board config not a printer config, please only state the pins and the offsets, it is up to the end user to properly set it for their touch probe.

Thanks
James

Reply:
Hi James, 

All done.

Kind regards,

ExplicitChimp